### PR TITLE
Release/1.5.0 alpha.3

### PR DESCRIPTION
**Pull request by agduncan94**
Updates for new release 1.5.0-alpha.3

### CODE REVIEW

File: dockstore-webservice/src/main/resources/swagger.yaml [hunk 1]
```diff
@@ -3267,7 +3267,7 @@ paths:
         schema:
           type: "array"
           items:
-            $ref: "#/definitions/SourceFile"
```
Comment (denis-yuen) [inline dockstore-webservice/src/main/resources/swagger.yaml]: Sigh

Comment (agduncan94) [reply dockstore-webservice/src/main/resources/swagger.yaml]: :sob:

Comment (garyluu) [reply dockstore-webservice/src/main/resources/swagger.yaml]: is this deterministically generated as `type: "object"` now?

Comment (denis-yuen) [reply dockstore-webservice/src/main/resources/swagger.yaml]: Don't know.
One other oddity,  editHostedTool and editHostedWorkflow got generated in different ways.

**Pull request merged**